### DISCUSSION TITLE
Clash Results Widget

### DISCUSSION THREAD

**Discussion by brinpat** [iTwin/community#6] [General]
Hi,

For the widget code in iTwin.js, it only seems to show one asset instead of both of the clashing assets. 

Also once a clash is selected for viewing it greys out all other assets and there is no way to return to the original view to see all assets again.

Is there a fix for this?

**Comment by prashanth-ananth** [comment]
Hi,
Just wanted to get more clarity.  Are you trying the code surfaced in the sandbox (https://www.itwinjs.org/sandboxes/iTwinPlatform/Clash%20Review).

You could try clearing it by using this code snippet.
provider.clearEmphasizedElements(vp);

https://www.itwinjs.org/reference/core-frontend/rendering/emphasizeelements/clearemphasizedelements/

HTH.

**Reply by brinpat** [reply]
Hi @prashanth-ananth yes that is the code I am using. I will try using the code snippet provider.clearEmphasizedElements(vp); and see how that does.

**Reply by brinpat** [reply]
How do I go about implementing that code snippet? I guess I would need to make a button too or make a click method to use the code.

**Comment by brinpat** [comment]
This has been fixed
